### PR TITLE
Show id_token in log

### PR DESCRIPTION
**Pull request by salmonz**
I noticed when I was following the example, I was watching for the id_token to appear in the log, as this is what is most important in the OIDC protocol, but only saw the access token. This patch shows the id_token for both manual code exchange and auto.

### CODE REVIEW

**Comment by WilliamDenniss** [comment]
Can you sign the [CLA](http://openid.net/intellectual-property/contribution-license-agreement/)?

**Comment by salmonz** [comment]
Signed

File: Examples/Example-iOS_Swift-Carthage/Source/AppAuthExampleViewController.swift [hunk 1]
```diff
@@ -396,6 +397,7 @@ extension AppAuthExampleViewController {
             if let authState = authState {
                 self.setAuthState(authState)
                 self.logMessage("Got authorization tokens. Access token: \(authState.lastTokenResponse?.accessToken ?? "DEFAULT_TOKEN")")
+                self.logMessage("Got id_token token. id_token: \(authState.lastTokenResponse?.idToken ?? "DEFAULT_TOKEN")")
```
Comment (WilliamDenniss) [inline Examples/Example-iOS_Swift-Carthage/Source/AppAuthExampleViewController.swift]: Can we handle the nil case better? Seems factually incorrect to state that we got an id_token when we didn't. I don't see how printing "DEFAULT_TOKEN" helps anyone. 

Same comment could apply to the accessToken, although I would normally expect it to be nonnull.

**Comment by WilliamDenniss** [comment]
Bulk closing old PRs, can re-open if there is activity.

**Pull request closed without merging**